cachito/workers/pkg_managers/rubygems.py

```diff
@@ -75,6 +75,9 @@ def parse_gemlock(source_dir, gemlock_path):
     dependencies = []
     all_gems = GemfileLockParser(str(gemlock_path)).all_gems
     for gem in all_gems.values():
+        if gem.version is None:
+            log.debug(f"RubyGem {gem.name} is without version, skipping.")
+            continue
```

Comment:
It would be ideal if the log message could explain a bit more about why it's skipping the dependency, otherwise users could be confused why Cachito behaves in this unexpected way